packages/usa.michigan/README.md

```diff
@@ -0,0 +1,112 @@
+# 🇺🇸 USA, Michigan
+
+[![License: CC BY SA 3.0](https://img.shields.io/badge/License-CC%20BY%20SA%203.0-blue.svg)](https://creativecommons.org/licenses/by-sa/3.0/)
+
+Contains all the counties of the Michigan state:
+* Alcona
+* Alger
+* Allegan
+* Alpena
+* Antrim
+* Arenac
+* Baraga
+* Barry
+* Bay
+* Benzie
+* Berrien
+* Branch
+* Calhoun
+* Cass
+* Charlevoix
+* Cheboygan
+* Chippewa
+* Clare
+* Clinton
+* Crawford
+* Delta
+* Detroit City
+* Dickinson
+* Eaton
+* Emmet
+* Genesee
+* Gladwin
+* Gogebic
+* Grand-Traverse
+* Gratiot
+* Hillsdale
+* Houghton
+* Huron
+* Ingham
+* Ionia
+* Iosco
+* Iron
+* Isabella
+* Jackson
+* Kalamazoo
+* Kalkaska
+* Kent
+* Keweenaw
+* Lake
+* Lapeer
+* Leelanau
+* Lenawee
+* Livingston
+* Luce
+* Mackinac
+* Macomb
+* Manistee
+* Marquette
+* Mason
+* Mecosta
+* Menominee
+* Midland
+* Missaukee
+* Monroe
+* Montcalm
+* Montmorency
+* Muskegon
+* Newaygo
+* Oakland
+* Oceana
+* Ogemaw
+* Ontonagon
+* Osceola
+* Oscoda
+* Otsego
+* Ottawa
+* Presque-Isle
+* Roscommon
+* Saginaw
+* Sanilac
+* Schoolcraft
+* Shiawassee
+* St-Clair
+* St-Joseph
+* Tuscola
+* Van-Buren
```

Comment:
Could you remove the dash ("-") of all the counties in the README and the SVG please? (e.g. "Grand-Traverse" -> "Grand Traverse", "St-Clair" -> "St. Clair"...)